01-understand-network/Starter/app/src/main/java/com/yourcompany/android/moviediary/MainActivity.kt

```diff
@@ -45,7 +45,7 @@ import androidx.compose.ui.Modifier
 import androidx.core.content.getSystemService
 import com.yourcompany.android.moviediary.networking.MovieDiaryApi
 import com.yourcompany.android.moviediary.ui.login.LoginScreen
-import com.yourcompany.android.moviediary.ui.movies.HomeScreen
```

Comment:
This change looks obsolete. I changed **home** package to **movies**, and `HomeScreen` to `MoviesScreen`.

Reply:
This is from an old commit, you should see the final changes here: https://github.com/kodecocodes/m3-nra-materials/pull/2/files